etc/kayobe/ansible/rabbitmq-reset.yml

```diff
@@ -26,46 +26,45 @@
         - "'synchronized: yes' not in timedatectl_status.stdout"
 
     - name: Inspect the {{ container_name }} container
-      shell:
-        cmd: "docker container inspect --format '{{ '{{' }} .State.Running {{ '}}' }}' {{ container_name }}"
+      ansible.builtin.shell:
+        cmd: docker container inspect --format '{{ '{{' }} .State.Running {{ '}}' }}' {{ container_name }}
       register: inspection
 
     - name: Ensure the {{ container_name }} container is running
-      command: "systemctl start kolla-{{ container_name }}-container.service"
+      ansible.builtin.command: systemctl start kolla-{{ container_name }}-container.service
       when: inspection.stdout == 'false'
 
     - name: Wait for the {{ container_name }} container to reach state 'Running'
-      shell:
-        cmd: "docker container inspect --format '{{ '{{' }} .State.Running {{ '}}' }}' {{ container_name }}"
+      ansible.builtin.shell:
+        cmd: docker container inspect --format '{{ '{{' }} .State.Running {{ '}}' }}' {{ container_name }}
       register: result
       until: result.stdout == 'true'
       retries: 10
       delay: 6
 
     - name: Wait for the rabbitmq node to automatically start on container start
-      command: "docker exec {{ container_name }} /bin/bash -c 'rabbitmqctl wait /var/lib/rabbitmq/mnesia/rabbitmq.pid --timeout 60'"
+      ansible.builtin.command: docker exec {{ container_name }} /bin/bash -c 'rabbitmqctl wait /var/lib/rabbitmq/mnesia/rabbitmq.pid --timeout 60'
       when: inspection.stdout == 'false'
 
     - name: Stop app
-      command: "docker exec {{ container_name }} /bin/bash -c 'rabbitmqctl stop_app'"
-
+      ansible.builtin.command: docker exec {{ container_name }} /bin/bash -c 'rabbitmqctl stop_app'
     - name: Force reset app
-      command: "docker exec {{ container_name }} /bin/bash -c 'rabbitmqctl force_reset'"
-
+      ansible.builtin.command: docker exec {{ container_name }} /bin/bash -c 'rabbitmqctl force_reset'
     - name: Start app
-      command: "docker exec {{ container_name }} /bin/bash -c 'rabbitmqctl start_app'"
-
+      ansible.builtin.command: docker exec {{ container_name }} /bin/bash -c 'rabbitmqctl start_app'
     - name: Wait for all nodes to join the cluster
-      command: "docker exec {{ container_name }} /bin/bash -c 'rabbitmqctl await_online_nodes {{ groups['controllers'] | length }}'"
-
+      ansible.builtin.command: docker exec {{ container_name }} /bin/bash -c 'rabbitmqctl await_online_nodes {{ groups['controllers'] | length }}'
```

Comment:
I'd prefer to keep the newlines between these tasks

Reply:
Same but idk if the linter agrees